openapi.yaml

```diff
@@ -114,7 +114,12 @@ paths:
         content:
           application/vnd.api+json:
             schema:
-              $ref: '#/components/schemas/EmployeeResource'
+              type: object
+              properties:
+                data:
+                  $ref: '#/components/schemas/EmployeeResponse'
```

Comment:
Not 100% sure we should be using a "response" object in the `PATCH`. I went down the path of creating a "patch body" but then decided that this is more inline with the original intent of the `EmployeeResource` being required.